Formula/s/scilla.rb

```diff
@@ -0,0 +1,18 @@
+class Scilla < Formula
+  desc "DNS, subdomain, port, directory enumeration tool"
+  homepage "https://github.com/edoardottt/scilla"
+  url "https://github.com/edoardottt/scilla/archive/refs/tags/v1.2.7.tar.gz"
+  sha256 "cccf86bc9c0ed70c2322d2921b06fa51905bdfb65ab51afe9c0df52411596cbb"
+  license "GPL-3.0-only"
```

Comment:
https://github.com/edoardottt/scilla/blob/961fbd26bfd24d63dbc6248f9c9543baaa3be160/pkg/input/keys.go#L1-L26
```suggestion
  license "GPL-3.0-or-later"
```